org.mamedev.MAME.metainfo.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.mamedev.MAME</id>
+  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
no project license?

Reply:
Added

Reply:
there can only be one `project_license`

Reply:
you can add multiple licenses separated by `OR` or `AND` in project license, as specified by spdx.

Reply:
ok.